en/news/_posts/2018-12-06-ruby-2-6-0-rc1-released.md

```diff
@@ -0,0 +1,138 @@
+
+  Ruby 2.5 improves block passing performance. [[Feature #14045]](https://bugs.ruby-lang.org/issues/14045)
+  Additionally, Ruby 2.6 improves the performance of passed block calling.
+  With micro-benchmark we can observe x2.6 improvement.
+
+* Transient Heap (theap) is introduced. [[Bug #14858]](https://bugs.ruby-lang.org/issues/14858) [[Feature #14989]](https://bugs.ruby-lang.org/issues/14989)
+  theap is managed heap for short-living memory objects which are pointed by
+  specific classes (Array, Hash, Object, and Struct). For example, making small
+  and short-living Hash object is x2 faster. With rdoc benchmark, we observed
+  6-7% performance improvement.
+
+## Other notable changes since 2.5
+
+* `$SAFE` is a process global state and we can set `0` again.  [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
+
+* Passing `safe_level` to `ERB.new` is deprecated. `trim_mode` and `eoutvar` arguments are changed to keyword arguments. [[Feature #14256]](https://bugs.ruby-lang.org/issues/14256)
+
+* Supported Unicode version is updated to 11. It is planed to update 12 and 12.1 in future TEENY releases of Ruby 2.6.
+
+* Merge RubyGems 3.0.0.beta3. `--ri` and `--rdoc` options was removed. Please use `--document` and `--no-document` options instead of them.
+
+* Merge [Bundler](https://github.com/bundler/bundler) as Default gems.
+
+See [NEWS](https://github.com/ruby/ruby/blob/v2_6_0_rc1/NEWS)
+or [commit logs](https://github.com/ruby/ruby/compare/v2_5_0...v2_6_0_rc1)
+for details.
+
+With those changes,
+[6376 files changed, 227364 insertions(+), 51599 deletions(-)](https://github.com/ruby/ruby/compare/v2_5_0...v2_6_0_rc1)
+since Ruby 2.5.0!
+
+Enjoy programming with Ruby 2.6.0-rc1!
+
+## Download
+
+* <https://cache.ruby-lang.org/pub/ruby/2.6/2.6.0-rc1.tar.bz2>
+
+      SIZE:   16823448 bytes
+      SHA1:   889db7731fd43f6dbf7f31ffdb0a29bba1a8d356
+      SHA256: 6d6183639ed9c02320d7132e97c65489a39e24d8b55fc4ed35ac53d1189cb61d
+      SHA512: ad101adee5c43f3645561e73970f15d4e9141f707da69a92e224575c665949e18ca53389e5315fca2ea393
+4d77967a59e304353cde4a915537e7c4e4ee20be73
+* <https://cache.ruby-lang.org/pub/ruby/2.6/2.6.0-rc1.tar.bz2>
+
+      SIZE:   20737499 bytes
+      SHA1:   457e39aee1978da5e42af42a6ad230421544aa07
+      SHA256: 2bcdf468de499e4d6983d60d63dcc883f4c54fdc05a08a54eb93d315477bc4cc
+      SHA512: 0842fae8a199f6c1e76f5d775edbf468e18a54f0419324eb73595e0268c728c71733371d71dc2fa342105dbc487987ca5556948a9ef067276a7b5f552462802a
+* <https://cache.ruby-lang.org/pub/ruby/2.6/2.6.0-rc1.tar.bz2>
+
+      SIZE:   14607078 bytes
+      SHA1:   269fe9d414d7731e4a63959fadffe5c50c08ce0e
+      SHA256: b4e9c0e8801946e9f0baba30948955f4341e9e04f363c206b7bd774208053eb5
+      SHA512: cbd6281b2aab6fbce3f699c1ab57e5423304dca7a547a0b3cd4e8e980326dc7b85b2ca2bfaf3f3a648d40f4222fdf1740d81d422790ee7ae1ba1ed33eb11e3e8
+* <https://cache.ruby-lang.org/pub/ruby/2.6/2.6.0-rc1.tar.bz2>
```

Comment:
These are all `.tar.bz2` links rather than `.tar.gz`, `.zip`, `.tar.bz2`, and `.tar.xz`.

Reply:
This is fixed by #1911, thanks!